### ISSUE TITLE
How to retrieve Parent orchestrations with Pagination?

### ISSUE THREAD

**Issue by Mohakane** [Azure/durabletask#733]
Hello, 

I want to query only the parent orchestrations with page size and page number ,
but I found out it's not implement it. (GetOrchestrationWithQueryAsync)

Do you plan to realize it ?

If yes, do you have any timeline ?

**Comment by Mohakane** [comment]
Hi @davidmrdavid,

We have a use case where we are supposed to list all parent instances on a table with pagination.
Once the user click on the instance it will redirect him to a different page showing the instance details and list also the sub Instances if any.

As I stated on first message I have noticed that GetOrchestrationWithQueryAsync doesn't allow me to specify to retrieve only the parent instances. I would say an instance is a parent if parentId is null.

if it's still not clear enough to you I can send you some code example to illustrate what I'm talking about.

Thanks.
